cromulent/vocab.py

```diff
@@ -376,23 +377,23 @@ def art2_post_init(self, **kw):
 		super(InformationObject, self)._post_init(**kw)
 	InformationObject._post_init = art2_post_init
 
-def add_proxy_wrapper():
```

Comment:
I don't understand why this is commented out. Just unused code?

Reply:
Yup, unused but potentially valuable in the future.

Reply:
Meh, not that valuable. Killed it.